src/resolvers/Mutations.js

```diff
@@ -142,9 +138,15 @@ function createOrganization(_, args, context, info){
         data: {
         nameEn: args.nameEn,
         nameFr: args.nameFr,
-        acronymEn: args.acronymEn,
-        acronymFr: args.acronymFr
-        }
+        acronymEn: args.acronymEn,        
+        acronymFr: args.acronymFr,
+        teams:{
+            create:{
+                nameEn: "",
```

Comment:
I see the names are set to empty on purpose, how will this present to users?

Reply:
Leaving it up to the front end to handle the name of the 'default team'.

Reply:
How will the frontend know it is a default team?  (Just rely on the empty name?)

Reply:
We hadn't really come up with a standard at the time.  We can set it to what ever we want really.